frontend/models/feedback-task-model/record/Record.model.js

```diff
@@ -0,0 +1,30 @@
+import { Model } from "@vuex-orm/core";
+import { RecordField as RecordFieldModel } from "../record-field/RecordField.model";
+import { RecordResponse as RecordResponseModel } from "../record-response/RecordResponse.model";
+
+class Record extends Model {
+  static entity = "records";
+  static primaryKey = ["record_id", "dataset_id"];
+
+  static fields() {
+    return {
+      id: this.uid(),
+      dataset_id: this.attr(null),
+      record_status: this.string(null).nullable(),
```

Comment:
This is a local status for the web app, right?

Reply:
this is the status of the record regarding the interaction of the user with the corresponding formular